NCS.DSS.EmploymentProgression/GetEmploymentProgression/Service/EmploymentProgressionGetTriggerService.cs

```diff
@@ -5,18 +5,18 @@ namespace NCS.DSS.EmploymentProgression.GetEmploymentProgression.Service
 {
     public class EmploymentProgressionGetTriggerService : IEmploymentProgressionGetTriggerService
     {
-        private readonly IDocumentDBProvider _documentDbProvider;
-        private readonly IServiceBusClient _serviceBusClient;
+        private readonly ICosmosDBProvider _cosmosDbProvider;
+        private readonly IEmploymentProgressionServiceBusClient _serviceBusClient;
 
-        public EmploymentProgressionGetTriggerService(IDocumentDBProvider documentDbProvider, IServiceBusClient serviceBusClient)
+        public EmploymentProgressionGetTriggerService(ICosmosDBProvider documentDbProvider, IEmploymentProgressionServiceBusClient serviceBusClient)
```

Comment:
Can we also rename this documentdbprovider as well (and make sure its name correctly in all files too)